contracts/Verifiers/TransferVerifier.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.7.6;
+
+import "../Libraries/ReplayBits.sol";
+import "../Libraries/TransferHelper.sol";
+
+/// @title Verifier for ETH and ERC20 transfers
+/// @notice These functions should be executed by metaDelegateCall() on Brink account proxy contracts
+contract TransferVerifier {
+
+  /// @dev Executes an ETH transfer with replay protection and expiry
+  /// @notice This should be executed by metaDelegateCall() with the following signed params
+  /// @param bitmapIndex The index of the replay bit's bytes32 slot
+  /// @param bit The value of the replay bit
+  /// @param recipient The recipient of the transfer
+  /// @param amount Amount of token to transfer
+  /// @param expiryBlock The block when the transfer expires
+  function ethTransfer(
+    uint256 bitmapIndex, uint256 bit, address recipient, uint256 amount, uint256 expiryBlock
+  )
+    external
+  {
+    require(expiryBlock > block.number, "EXPIRED");
```

Comment:
Nitpick, but should this be >=

Reply:
I think it should be `>` since "expiryBlock" implies the block when the message expires. if it was `>=` then it would expire in `expiryBlock + 1` instead of in `expiryBlock`